README.md

```diff
@@ -83,4 +83,5 @@ But what if you don't know what kind of website to "just build"? The following i
 * Create a custom 404 page
 * Create a random number generator for picking lottery numbers ([Guidelines](docs/lottery-numbers.md))
 * Go to the [new repo page](https://github.com/new) on Github. Make something inspired by the randomly-generated repo name they give you
-* Recreate the title sequence of a favorite show or movie using CSS. (Example: [Stranger Things](https://github.com/wbobeirne/stranger-things))
\ No newline at end of file
+* Recreate the title sequence of a favorite show or movie using CSS. (Example: [Stranger Things](https://github.com/wbobeirne/stranger-things))
+* Create a frontend with historic or futuristic theme (or any theme accordingly).
```

Comment:
Instead of frontend, maybe say user interface? Or you could rephrase as "Create a site theme with historic or futuristic influence"?

Reply:
Yes Sure :)